os/include/tinyara/pm/pm.h

```diff
@@ -588,6 +588,22 @@ int pm_changestate(enum pm_state_e newstate);
 
 enum pm_state_e pm_querystate(void);
 
+/****************************************************************************
+ * Name: pm_recover
+ *
+ * Description:
+ *   This function is called from task_recover() when a task is deleted via
+ *   task_delete() or via pthread_cancel(). It do pm post cleaning.
+ *
+ * Inputs:
+ *   tcb - The TCB of the terminated task or thread
+ *
```

Comment:
@gSahitya-samsung use code-i feature to generate the doxzgen documentation for the interfaces and update the description if it not suiting better.(/docstring) which is very productive.